docker/Dockerfile

```diff
@@ -360,7 +360,7 @@ COPY --from=git_clone "/tmp/morpheus_repo/conda/environments/*.yaml" "./conda/en
 COPY --from=git_clone "/tmp/morpheus_repo/docker" "./docker"
 COPY --from=build_docs "/workspace/${MORPHEUS_ROOT_HOST}/build/docs/html" "./docs"
 COPY --from=git_clone "/tmp/morpheus_repo/examples" "./examples"
-COPY --from=git_clone "/tmp/morpheus_repo/models/datasets" "./models/datasets"
+COPY --from=git_clone "/tmp/morpheus_repo/models" "./models"
```

Comment:
Remove this entire line and move the log-parsing and fraud detection files necessary to run examples to the `examples/data` folder